tripal/tests/src/Kernel/TripalImporter/TripalImporterFormBuildTest.php

```diff
@@ -200,6 +201,13 @@ public function testTripalImporterForm() {
     // a submit button.
     $this->assertArrayHasKey('button', $form,
       "The form should have a submit button since we indicated a specific importer.");
+    // Do not publish checkboxes.
```

Comment:
I'm confused by this comment. Maybe you meant

```suggestion
    // Publish checkboxes.
```